src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -278,6 +282,10 @@ export class DataViewsService {
    * Can the user save data views?
    */
   public getCanSave: () => Promise<boolean>;
+  /**
+   * Can the user save advanced settings?
+   */
+  public getCanSaveAdvancedSettings: () => Promise<boolean>;
```

Comment:
This should be private, as its only use is internal to getDefaultDataView

Reply:
@mattkime I did but now TypeScript is complaining, can you help

```
property 'getCanSaveAdvancedSettings' is private in type 'DataViewsServicePublic' but not in type 'DataViewsServicePublic'.
```